### PR TITLE
Docs: Enhance create_changelog_view usage

### PR DESCRIPTION
**Pull request by manuzhang**

### CODE REVIEW

**Comment by manuzhang** [comment]
@nastra @flyrain please help review this PR.

File: docs/docs/spark-procedures.md [hunk 1]
```diff
@@ -823,6 +823,8 @@ second snapshot deleted 1 record.
 |2	| Bob	   |INSERT	|0	|5390529835796506035|
 |1	| Alice  |DELETE	|1	|8764748981452218370|
 
+#### Net Changes
```
Comment (flyrain) [inline docs/docs/spark-procedures.md:826]: This is an item under the subtitle `example`, it should not be the same level as `example`. Can we change it to one level down, like `##### Net Changes`?

File: docs/docs/spark-procedures.md [hunk 2]
```diff
@@ -756,8 +756,8 @@ Creates a view that contains the changes from a given table.
...
 | `options`            |           | map<string, string> | A map of Spark read options to use                                                                                                                                                                   |
-| `net_changes`        |           | boolean             | Whether to output net changes (see below for more information). Defaults to false.                                                                                                                   |
-| `compute_updates`    |           | boolean             | Whether to compute pre/post update images (see below for more information). Defaults to false.                                                                                                       | 
+| `net_changes`        |           | boolean             | Whether to output net changes (see below for more information). Defaults to false. It must be false when `compute_updates` is true.                                                                                                                  |
```
[the remark this answers is not in the document]
Comment (manuzhang) [reply docs/docs/spark-procedures.md:759]: It's not disabled but throws error 'Not support net changes with update images' when both are true.

File: docs/docs/spark-procedures.md (hunk 1 above)
Comment (manuzhang) [reply docs/docs/spark-procedures.md:826]: I think `Net Changes` is a feature like `Pre/Post Update Images` so they should be on the same level.

File: docs/docs/spark-procedures.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (flyrain) [reply docs/docs/spark-procedures.md:759]: Fair enough. Let's keep it as is.

File: docs/docs/spark-procedures.md (hunk 1 above)
Comment (flyrain) [reply docs/docs/spark-procedures.md:826]: I'm OK to bring it up to the same level as `Pre/Post Update Images`. By doing that, we will need to give a brief introduction before jumping into an example.  Something like this:
```
The procedure can remove intermediate changes across multiple snapshots, and only outputs the net changes. Here is an example to create a changelog view that computes net changes. 
```

**Review by flyrain — APPROVED** [review]
LGTM. Thanks for working on it.

**Comment by manuzhang** [comment]
@flyrain Thanks for review.

**Pull request merged**